articles/cognitive-services/Speech-Service/quickstarts/text-to-speech/async-synthesis-long-form-audio.md

```diff
@@ -14,7 +14,7 @@ ms.author: erhopf
 
 # Quickstart: Asynchronous synthesis for long-form audio in Python (Preview)
 
-In this quickstart, you'll use the Long Audio API to asynchronously convert text to speech, and retrieve the audio output from a URI provided by the service. This REST API is ideal for content providers that need to convert text files greater than 10,000 characters or 50 paragraphs into synthesized speech. For more information, see [Long Audio API](../../long-audio-api.md).
+In this quickstart, you'll use the Long Audio API to asynchronously convert text to speech, and retrieve the audio output from a URI provided by the service. This REST API is ideal for content providers that need to synthysize audio more than 10 minutes. For more information, see [Long Audio API](../../long-audio-api.md).
```

Comment:
Hi @Yueying-Liu , how does someone know that the text file is longer than ten minutes? This isn't clear to me, whereas, the characters are something that they can measure.

Reply:
This REST API is ideal for content providers that need to synthesize audio more than 5,000 characters

Reply:
Let's update the text to say that, since it's hard for a first time user to determine how much text makes up 10 minutes.

Reply:
As you've stated below. Let's use the 5,000 characters, and we can state that it's approximately 10 minutes of audio.